apps/examples/power/power_main.c

```diff
@@ -289,6 +310,13 @@ int power_main(int argc, char *argv[])
 
 		is_running = false;
 
+		pid = task_create("stop_pm_test", 100, 1024, stop_pm_test, argv + 1);
```

Comment:
Could you let me know why you give `argv + 1`?
In help message, the stop command does not have any following argument.

And the task creation for start also give it, but is it right with `+1`? not `+2`?

Reply:
@seokhun-eom24 You already check the argv[1] with start.
```
	if (strncmp(argv[1], "start", 6) == 0) {
		if (is_running) {
			printf("power test is already running\n");
			return 0;
		}
		is_running = true;
		pid = task_create("start_pm_test", 100, 1024, start_pm_test, argv + 1);
```
The task is created for start. Why do you need to give `start` argument to the task?